internal/domain/tdexFeeder.go

```diff
@@ -0,0 +1,65 @@
+package domain
+
+import (
+	"errors"
+	"sync"
+)
+
+type TdexFeeder interface {
+	Start() error
+	Stop()
+	IsRunning() bool
+}
+
+type tdexFeeder struct {
```

Comment:
This does not seem to me a business domain, rather than a `service` that should belong to the application layer wich manipulate domains.